modules/ui/src/app/app.component.ts

```diff
@@ -326,7 +326,8 @@ export class AppComponent implements AfterViewInit {
         take(1)
       )
       .subscribe(() => {
-        this.appStore.setFocusOnPage(null);
+        const mainContainer = window.document.querySelector('#main');
+        this.appStore.setFocusOnPage(mainContainer);
```

Comment:
Maybe we don't need to pass any argument here?  
In focus-manager.service the container will be #main in this case
  focusFirstElementInContainer(
    container: Document | Element | null = window.document.querySelector(
      '#main'
    )
  )

Reply:
> Maybe we don't need to pass any argument here? In focus-manager.service the container will be #main in this case focusFirstElementInContainer( container: Document | Element | null = window.document.querySelector( '#main' ) )

Right, I saw it, but it doesn't work.